app/src/main/res/layout/event_layout.xml

```diff
@@ -17,7 +17,7 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="?attr/selectableItemBackground"
+        android:background="@{ event.state.equals(`draft`) ? @color/event_state_draft : 0 }"
```

Comment:
This will again make the app lose ripple effect

Reply:
I have added the `selectableItemBackground` on line 39. As far as I remember, it was working when I checked. I will check once more.

Reply:
But `0` is a magic number, why is that used?

Reply:
It seemed to have no effect. And I read (saw) somewhere someone using this. Sorry for being silly, but I think data binding doesn't support `only if statement`. Or if any other solution you suggest.

Reply:
Use `selectableItemBackground` instead